src/WooCommerce.php

```diff
@@ -378,7 +378,8 @@ public static function getOrderDetailsHtmlDataAttr($order_id) {
     $order_data = $order->get_data();
 
     $order_details = [
-      'id' => $order->get_order_number(),
+      'order_number' => $order->get_order_number(),
+      'order_id' => $order_id,
```

Comment:
As far as I understand the code, the `$order_id` appears to be the internal post ID of the WooCommerce order. I would have expected to see only the "public identifier of the order" in the tracking – much like we only deal with product SKUs and not the internal IDs.

```suggestion
      'order_id' => $order->get_order_number(),
```

I don't know whether my understanding is correct though.  In case it isn't, then that would be a sign for missing documentation.

Reply:
What I was trying to do here is to prevent and order to be tracked by post_id after it has already been tracked by order_id. Anyway, Daniel Schmidt just commented on the ticket that he is checking a GA filter to exclude orders tracked by post_id.